cmd/alleleSplit/toSam.go

```diff
@@ -93,5 +93,13 @@ func SnpSearch(samfile string, genotypeVcf string, fOne string, parentOne string
 			sam.WriteAlnToFileHandle(childTwo, read)
 		}
 	}
-	reader.File.Close()
+}
+
+func getOldSeq(v *vcf.Vcf) [][]dna.Base {
```

Comment:
Maybe getSeq?  I think the "Old" part makes sense to you because you are thinking about the previous version of the structs, but I think it may be confusing to other people.